pages/axes.py

```diff
@@ -5,21 +5,23 @@
 
 dash.register_page(
     __name__,
-    description="Interactive Plotly Axes Labels",
+    description="Interactive Plotly Axes Labels example in a bar plot. This app has a top-bottom layout and a regular-callback.",
 )
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
 
-#### Plotly Documentation:  
+#### Dash Components in App:  
+- [Dcc.Dropdown](https://dash.plotly.com/dash-core-components/dropdown)
 
+#### Plotly Documentation:  
 - [Adjusting the axes properties](https://plotly.com/python/axes/)
+- [Bar Plot](https://plotly.com/python/bar-charts/)
```

Comment:
although it looks like a bar plot, the code is actually using a px.histogram. Can you please replace this.